HeroHours/admin.py

```diff
@@ -28,13 +29,16 @@ def check_out(modeladmin, request, queryset):
     updated_log = []
     for user in getall:
         lognew = models.ActivityLog(
-            userID=user.User_ID,
+            user=models.Users.objects.filter(User_ID=user.User_ID).first(),
```

Comment:
this looks to me like a extra repeated db query. Activity log should still have a user_id property that's made automatically by django. Could you use that instead?